linux-compose.yml

```diff
@@ -0,0 +1,19 @@
+version: "3.9"
+
+services:
+  upgrader-sdk:
+    image: spryker-sdk/upgrader-local:latest
+    build:
+      context: ./
+      dockerfile: infrastructure/dev.Dockerfile
+      args:
+          SPRYKER_PARENT_IMAGE: spryker/php:7.4
+    environment:
+        SPRYKER_XDEBUG_HOST_IP: ${SPRYKER_XDEBUG_HOST_IP:-host.docker.internal}
+    tty: true
+    command: "/bin/bash"
+    volumes:
+       - ~/.ssh:/root/.ssh
+       - ./:/data
+       - ../b2c-demo-shop:/data/project
```

Comment:
why do we hardcode it?

Reply:
Just as an example. From my perspective, it helps developers to understand how it should work

Reply:
I suppose it will be better to describe it in the "Local Development" section of the documentation

Reply:
Thanks, did it